### ISSUE TITLE
Unable to resume training from check point with custom loss function in NBEATS

### ISSUE THREAD

**Issue by AndresMR77** [unit8co/darts#1758]
**Describe the bug**

I am training an NBEATS model with a custom loss function defined by myself. I train the model (set save_checkpoints = True) for a certain amount of epochs and then I load it again (load_from_checkpoint) to continue training, but when I do so, the loss function switches to the default MSELoss instead of continuing with my custom loss.

**System (please complete the following information):**
 - Python version: [e.g. 3.7,13]
 - darts version [e.g. 0.22.0]
